src/uct/tcp/tcp.h

```diff
@@ -255,6 +255,7 @@ struct uct_tcp_ep {
     uint8_t                       flags;            /* Endpoint flags */
     uct_tcp_ep_conn_state_t       conn_state;       /* State of connection with peer */
     int                           fd;               /* Socket file descriptor */
+    int                           accept_fd;        /* Old socket file descriptor */
```

Comment:
1. IMO name is confusing, maybe old_fd / prev_fd / ghost_fd 
/* Old file descriptor which should be close as soon as we get a connection
  request on the new fd */

2. Maybe reduce "retries" to u8/u16 and move it after conn_state to not increase ep size

Reply:
1. renamed to `stale_fd` as discussed offline
2. reduced to `uint8_t` and move it before `conn_state`

the socket fd on EP created from API on "loser" simultaneous connection side will be closed when sending ACK + REQ, since we can don't care about this connection (REQ will be sent anyway as a part of ACK packet).